app/components/service/service.html

```diff
@@ -72,6 +72,13 @@
                 <input type="text" class="form-control" ng-model="service.Image" ng-change="updateServiceAttribute(service, 'Image')" ng-disabled="isUpdating" />
               </td>
             </tr>
+            <tr>
```

Comment:
We need to add a condition on this, only display this section for API version >= 1.30